test.md

```diff
@@ -0,0 +1 @@
+

```

Comment:
could please put the "test.md" file inside the gitignore file, it should be ignored